proto/kava/earn/v1beta1/proposal.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+package kava.earn.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/kava-labs/kava/x/earn/types";
+
+// CommunityPoolDepositProposal deposits from the community pool into an earn vault
+message CommunityPoolDepositProposal {
+  option (gogoproto.goproto_stringer) = false;
+  option (gogoproto.goproto_getters)  = false;
+
+  string                       title          = 1 [(gogoproto.moretags) = "yaml:\"title\""];
```

Comment:
I believe we don't need the yaml tags anymore, the `gogoproto.moretag` options can be removed